pkg/registries/ecr/ecr_test.go

```diff
@@ -42,7 +42,6 @@ func TestECRIntegration(t *testing.T) {
 	require.NoError(t, err)
 
 	assert.NoError(t, ecr.Test())
-	assert.Equal(t, fmt.Sprintf("%s.dkr.ecr.us-west-2.amazonaws.com", registryID), ecr.endpoint)
```

Comment:
this isn't still true?

Reply:
It is still true, I removed it because `ecr.endpoint` no longer exists. But we can add it back if we assert the endpoint from the config.